native/acter/src/api/verification.rs

```diff
@@ -1077,11 +1127,9 @@ impl SessionManager {
 }
 
 impl Client {
-    pub fn verification_event_rx(&self) -> Option<Receiver<VerificationEvent>> {
-        match self.verification_controller.event_rx.try_lock() {
-            Ok(mut r) => r.take(),
-            Err(e) => None,
-        }
+    pub fn verification_event_rx(&self) -> impl Stream<Item = OptionVerificationEvent> {
+        BroadcastStream::new(self.verification_controller.event_rx.resubscribe())
+            .map(|o| OptionVerificationEvent::new(o.ok()))
```

Comment:
Do I understand correctly that this entire new type and the wrapping and all is due to that event sometimes being `None`? Wouldn't `.filter_map` do enough for us here?

```suggestion
    pub fn verification_event_rx(&self) -> impl Stream<Item = VerificationEvent> {
        BroadcastStream::new(self.verification_controller.event_rx.resubscribe())
            .filter_map(|o| o.ok())
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2411/commits/8ff8191aad386d987fffd4a938a9d636fe9efbcc